### ISSUE TITLE
PB menu drop-down menu does not display info for Firefox Private windows

### ISSUE THREAD

**Issue by wsgibson** [EFForg/privacybadger#1877]
For all Firefox Private windows/tabs only:

1. The PB menu button displays the correct blockage counts.
2. The PB drop-down menu does not display the version (only says "version ").
3. The PB drop-down menu has no tracking information displayed when it is blocking trackers (the menu view looks identical to when no trackers are being blocked by PB).

Looking at previous issues I tried removing the PB menu button and adding it back but I still have the same issue.

I am running Firefox Quantum 58.0.2 64-bit on a Mac OS (High Sierra).

**Comment by ghostwords** [comment]
Duplicate of #1060.
